views/user/logged-out.html

```diff
@@ -0,0 +1,4 @@
+{% extends "base.html" %}
+{% block content %}
+<p>You have been logged out! Come back soon.</p>
```

Comment:
Looks like this template isn't used. A flash message on home page would be great.